src/saml20_clj/coerce.clj

```diff
@@ -268,6 +282,46 @@
   (->SAMLObject [this]
     (->SAMLObject (->Element this))))
 
+(extend-protocol CoerceToLogoutResponse
+  nil
+  (->LogoutResponse [_] nil)
+
+  org.opensaml.saml.saml2.core.LogoutResponse
+  (->LogoutResponse [this] this)
+
+  org.opensaml.saml.common.SignableSAMLObject
+  (->LogoutResponse [this]
+    (throw (ex-info (format "Don't know how to coerce a %s to a Response" (.getCanonicalName (class this)))
```

Comment:
this pattern is strange to me. Why is the `throw` inside of a `->LogoutResponse` ? Does this make a nicer stacktrace?